backend/inspirehep/records/indexer/base.py

```diff
@@ -62,6 +62,8 @@ def _process_bulk_record_for_index(
         index_from_record, doc_type_from_record = self.record_to_index(record)
         if not index:
             index = index_from_record
+
+        index, doc_type = self._prepare_index(index, doc_type or doc_type_from_record)
```

Comment:
why we need the _prepare_index? :/

Reply:
Because _prepare_index is building index name with prefix etc: https://github.com/inveniosoftware/invenio-indexer/blob/8d35a88b79c9008e2ff5ecbff0c74ec915e428e4/invenio_indexer/api.py#L318-L320
I tried not to replace main logic from Invenio-search to be as much compatible as possible.